tests/unit/test_bake.py

```diff
@@ -141,9 +141,19 @@ def test_gpcp_bake(
             else:
                 assert job_name.startswith("gh-pforgetest-gpcp-from-gcs-")
 
+            # In beam-refactor, the actual zarr store is produced in a
+            # *subpath* of target_storage.rootpath, rather than in the
+            # root path itself. This is a compatibility break vs the previous
+            # versions of pangeo-forge-recipes. https://github.com/pangeo-forge/pangeo-forge-recipes/pull/495
+            # has more information
+            if recipes_version_ref == "beam-refactor":
+                zarr_store_path = config["TargetStorage"]["root_path"] + "/gpcp/"
```

Comment:
```suggestion
                zarr_store_path = config["TargetStorage"]["root_path"] + "gpcp/"
```
@yuvipanda, looks like the appended subpath here does not need a leading `/` because the root_path already terminates in `/`. We can see in the [failing test output here](https://github.com/pangeo-forge/pangeo-forge-runner/actions/runs/4273344225/jobs/7439196562?pr=64#step:7:83), that the bucket key is being resolved with a double slash:

```
api_params = {'Bucket': 'gpcp', 'Key': 'target//gpcp/.zgroup'}
```
The error does appear to be an s3 related so this might be all it is. I'll commit this and we can see if it fixes the problem.